tests/ProtoTests/ProtoTest.php

```diff
@@ -47,8 +47,14 @@ private function runProtoTest(string $protoPath): void
         // TODO: Check that all expected files are actually generated!
     }
 
-    public function testProtoBasic(): void
+    public function testBasic0(): void
```

Comment:
nit: there's a 0 at the end of this method name?

Reply:
Yes :) The `0` is there to make this test name _not_ be a prefix of other test names, meaning it's easy to run just this test!
The same is done in the C# tests.